pkg/odo/cli/application/application.go

```diff
@@ -333,17 +341,17 @@ func printDeleteAppInfo(client *occlient.Client, appName string) error {
 	}
 
 	for _, currentComponent := range componentList {
-		_, _, componentURL, appStore, err := component.GetComponentDesc(client, currentComponent.Name, appName)
+		componentDesc, err := component.GetComponentDesc(client, currentComponent.Name, appName)
 		if err != nil {
 			return errors.Wrap(err, "unable to get component description")
 		}
 		fmt.Println("Component", currentComponent.Name, "will be deleted.")
 
-		if len(componentURL) != 0 {
+		if len(componentDesc.URList) != 0 {
 			fmt.Println("  Externally exposed URL will be removed")
```

Comment:
Should that be URL(s) instead? Can a component have multiple URLs actually?

Reply:
So good thing we're not just outputting the first one anymore! ;)

Reply:
done